wordpress-mobile/WordPress-Android
# [WIP] Self hosted user detail

This PR adds a screen for the self-hosted user list. As with #21264, dummy data is being used here. Actual data requested via wordpress-rs will come later.

To test:

* Enable `self_hosted_users` in debug settings
* Switch to a self-hosted site
* Tap "Users" on the My Site page
* Tap a user in the list
* Verify the detail appears correctly
* If the user has an avatar in the detail screen, tap it and verify it's shown in a larger view

![detail](https://github.com/user-attachments/assets/2d59ccca-3525-4b2f-a969-ab274728f9b8)
